src/routing_membrane.rs

```diff
@@ -1716,7 +1869,7 @@ fn populate_routing_node() -> RoutingMembrane<TestInterface> {
         assert_eq!(Tester::new().call_operation(put_data_response,
             SourceAddress::Direct(Random::generate_random()),
             DestinationAddress::Direct(Random::generate_random()),
-            Authority::NaeManager(Random::generate_random())).call_count, 1usize);
+            Authority::NaeManager(Random::generate_random())).call_count, 0usize);
```

Comment:
If sentinel is doing what it's supposed to we would expect the return to be zero given the current setup of the test. Presumably additional or updated tests can still be addressed with MAID-1265.